src/openMVG/types.hpp

```diff
@@ -28,19 +28,15 @@ typedef std::pair<IndexT,IndexT> Pair;
 typedef std::set<Pair> Pair_Set;
 typedef std::vector<Pair> Pair_Vec;
 
-#define OPENMVG_NO_UNORDERED_MAP 1
-
-#if defined OPENMVG_NO_UNORDERED_MAP
+#if defined OPENMVG_UNORDERED_MAP
```

Comment:
at this point i think we can just and safely remove those #if as c++11 is now required to compile openmvg.